packages/autometrics/src/exporter-prometheus/mod.ts

```diff
@@ -0,0 +1,66 @@
+import { MetricReader } from "npm:@opentelemetry/sdk-metrics@^1.17.0";
+
+import {
+  BuildInfo,
+  amLogger,
+  createDefaultBuildInfo,
+  recordBuildInfo,
+  registerExporter,
+} from "../../mod.ts";
+import { PrometheusExporter } from "./PrometheusExporter.ts";
+
+let metricReader: MetricReader | undefined;
+
+export type InitOptions = {
+  /**
+   * Optional build info to be added to the `build_info` metric.
+   */
+  buildInfo?: BuildInfo;
+
+  /**
+   * Hostname or IP address on which to listen.
+   *
+   * @default '0.0.0.0'
+   */
+  hostname?: string;
+
+  /**
+   * Port on which to open the Prometheus scrape endpoint (default: 9464).
+   */
+  port?: number;
+};
+
+/**
+ * Initializes and registers the Prometheus exporter for Autometrics.
+ *
+ * This opens up a webserver with the `/metrics` endpoint, to be scraped by
+ * Prometheus.
+ */
+export function init({
+  buildInfo,
+  hostname = "0.0.0.0",
+  port = 9464,
+}: InitOptions = {}) {
+  if (metricReader) {
+    throw new Error("Prometheus exporter is already running");
```

Comment:
```suggestion
    throw new Error("Prometheus exporter is already running. Check if you might have run the init() function twice or you're attempting to use multiple exporters (only 1 is allowed)");
```

I understand this would throw in the above conditions so made the error a little bit clearer.

Reply:
I haven't tested it, but we *should* actually support multiple exporters now :) I'll add the message about calling `init()` twice though :+1: